content/en/docs/release-notes/release-notes-1.3.md

```diff
@@ -21,7 +40,57 @@ The zone is now `<Application Name>\<Issuing Template Alias>`
 
 [Venafi Cloud Issuer]: https://cert-manager.io/docs/configuration/venafi/
 
+## cert-manager controller
+
+The `--renew-before-expiration-duration` flag has been removed from the cert-manager controller, having been deprecated in the previous release.
+
+## cert-manager CRDs
+
+`CertificateRequests` are now immutable - the `spec` and `metadata.annotations` fields cannot be changed after creation. They were always designed to be immutable but this behavior is now *enforced* by the cert-manager webhook.
+
 # New Features
 
+## Upstream CSR Support Groundwork
+
+* The [design documentation](https://github.com/jetstack/cert-manager/blob/v1.3.0/design/20210203.certificate-request-identity.md) for Certificate Identity is now available.
+* `CertificateRequests` now have identity fields mirroring the upstream [Kubernetes CSR](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/) object.
+* `CertificateRequests` are now immutable.
+* `CertificateRequests` now have an Approval condition type, with `Approved` and `Denied` reasons.
+* The cert-manager controller currently always approves any `CertificateRequest`.
+* Added `kubectl cert-manager [approve|deny]` commands to the kubectl plugin.
+
+## cert-manager CRDs
+
+* `CertificateRequests` now support the `revisionHistoryLimit` field to limit the amount of retained history. The default is unlimited (`nil`).
+
+## Vault Enterprise
+
+* cert-manager now sends the `X-VAULT-NAMESPACE` header for the `requestTokenWithAppRoleRef` API call.
 
 # Bug Fixes
+
+## cert-manager controller
+
+* Fixed an issue which could cause multiple `CertificateRequests` to be created in a short time for a single `Certificate` resource.
+* Certificate Readiness controller only updates a certificate's status if something has changed.
+
+## selfSigned Issuer
+
+* The issuer now warns if you request a certificate with an empty subject DN - creating a certificate that is in violation of RFC 5280 and rejected as malformed by Java.
```

Comment:
`Java` is many things, do you mean `java.security.cert.Certificate#verify` or some other library/method?

Reply:
I honestly don't know "Java" was reported in https://github.com/jetstack/cert-manager/issues/3634

Reply:
Looks like the stack trace says `java.security.cert.CertificateParsingException`